packages/app/src/main.ts

```diff
@@ -3,7 +3,7 @@ import './main.scss'
 import 'virtual:windi.css'
 import urql from '@urql/vue'
 import App from './App.vue'
-import { makeUrqlClient } from './graphql/urqlClient'
+import { makeUrqlClient } from '@packages/frontend-shared/src/graphql/urqlClient'
```

Comment:
I cannot get barreling to work properly; it always breas `yarn build-prod`. If anyone knows how to fix this, please let me know... ideally we'd just do `import { makeUrqlClient } from '@packages/frontend-shared'` or something.